Assets/GothicVR/Scripts/Creator/VobCreator.cs

```diff
@@ -222,6 +223,16 @@ private static GameObject LoadVob(IVirtualObject vob, GameObject parent = null)
                     break;
                 }
                 case VirtualObjectType.oCMobInter:
+                    {
+                        if(vob.Name.ContainsIgnoreCase("bench") ||
+                            vob.Name.ContainsIgnoreCase("chair") ||
+                            vob.Name.ContainsIgnoreCase("throne"))
+                        {
+                            go = CreateSeat(vob, parent);
+                            _cullingVobObjects.Add(go);
+                        }
+                        break;
```

Comment:
We need to create a default mesh if it's not a seat

Reply:
of course! thanks for checking :)